Vagrantfile

```diff
@@ -0,0 +1,43 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# Author: David Manouchehri
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "bento/ubuntu-16.04"
+
+	config.vm.synced_folder ".", "/vagrant", disabled: true
+
+	config.vm.provision "shell", inline: <<-SHELL
+		apt-get update
+		# DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+		DEBIAN_FRONTEND=noninteractive apt-get -y install ccache cmake make g++-multilib gdb pkg-config realpath python-pexpect manpages-dev git ninja-build capnproto libcapnp-dev
+		apt-get clean
+	SHELL
+
+	config.vm.provision "shell", privileged: false, inline: <<-SHELL
+		git clone https://github.com/mozilla/rr.git
+		cd rr
+		mkdir obj
+		cd obj
+		cmake ..
+		make -j8
+		make test
+	SHELL
+
+	config.vm.provision "shell", inline: <<-SHELL
+		cd /home/vagrant/rr/ojb/
```

Comment:
Actually, `obj` doesn't look right

Reply:
Oops, fat fingered that right before committing.